docs/git-comparison.md

```diff
@@ -32,9 +32,9 @@ various use cases.
   branch', and new commits will automatically update the branch. This is
   necessary in Git because Git might otherwise lose track of the new commits.
   Jujutsu does not have a 'current branch'; instead, you update branches
-  manually. For example, if you check out a commit with a branch, new commits
-  are created on top of the branch, then you issue a later command to update the
-  branch.
+  manually. For example, if you start work on top of a commit with a branch,
```

Comment:
cc @martinvonz, does this work for you? As you haven't had a clear idea on how we should replace the "check out" verb.

Reply:
SGTM

Reply:
thanks